src/test/java/com/marizueva/laboratory/hw5/steps/JdiSiteDifferentElementPageSteps.java

```diff
@@ -0,0 +1,124 @@
+    public void selectYellowDropdown() {
+        differentElementPage
+                .chooseYellowDropdown();
+
+        updateLogList();
+    }
+
+    @Then("separate log for Water checkbox in log section with true status")
+    public void checkboxWaterInTrueStatus() {
+        List<String> expectedLogs = TestContext.getInstance()
+                .getTestObject(TestContext.ALL_LOGS_FROM_DIFFERENT_ELEMENT_PAGE);
+
+        List<WebElement> actualLogs = differentElementPage
+                .getAllLogs();
+        List<String> actualLogsInStringFormat = allLogsWithoutDate(actualLogs);
+
+        assertTrue(expectedLogs.containsAll(actualLogsInStringFormat));
+        assertTrue(actualLogsInStringFormat.contains(UsedInTestTerms.diffElWaterTrueLog));
+    }
+
+    @Then("separate log for Wind checkbox in log section with true status")
+    public void checkboxWindInTrueStatus() {
+        List<String> expectedLogs = TestContext.getInstance()
+                .getTestObject(TestContext.ALL_LOGS_FROM_DIFFERENT_ELEMENT_PAGE);
+
+        List<WebElement> actualLogs = differentElementPage
+                .getAllLogs();
+        List<String> actualLogsInStringFormat = allLogsWithoutDate(actualLogs);
+
+        assertTrue(expectedLogs.containsAll(actualLogsInStringFormat));
+        assertTrue(actualLogsInStringFormat.contains(UsedInTestTerms.diffElWindTrueLog));
+    }
+
+    @Then("separate log for Selen radio in log section with true status")
+    public void radioSelenInTrueStatus() {
+        List<String> expectedLogs = TestContext.getInstance()
+                .getTestObject(TestContext.ALL_LOGS_FROM_DIFFERENT_ELEMENT_PAGE);
+
+        List<WebElement> actualLogs = differentElementPage
+                .getAllLogs();
+        List<String> actualLogsInStringFormat = allLogsWithoutDate(actualLogs);
+
+
+        System.out.println(expectedLogs.toString());
+        System.out.println(actualLogsInStringFormat.toString());
+        assertTrue(expectedLogs.containsAll(actualLogsInStringFormat));
+        assertTrue(actualLogsInStringFormat.contains(UsedInTestTerms.diffElChoosenSelenLog));
+    }
+
+    @Then("separate log for Yellow dropdown in log section with true status")
+    public void yellowDropdownInTrueStatus() {
+        List<String> expectedLogs = TestContext.getInstance().getTestObject(TestContext
+                .ALL_LOGS_FROM_DIFFERENT_ELEMENT_PAGE);
+
+        List<WebElement> actualLogs = new DifferentElementPage(TestContext.getInstance()
+                .getTestObject(TestContext.WEB_DRIVER))
+                .getAllLogs();
+        List<String> actualLogsInStringFormat = allLogsWithoutDate(actualLogs);
+
+        assertTrue(expectedLogs.containsAll(actualLogsInStringFormat));
+        assertTrue(actualLogsInStringFormat.contains(UsedInTestTerms
+                .diffElDropdownYellowLogInTrue));
+    }
+
+    private void updateLogList() {
+        List<WebElement> allLogs = new DifferentElementPage(TestContext.getInstance()
```

Comment:
Why do you create new instance of the `DifferentElementPage` here instead of using `differentElementPage`?

Reply:
fixed